File: alexis.php
<!-- i dont actually know php im just winging it -->

<?php
class Alexis {
  public $name; 
  public $swag;

  public function __construct() {
    $name = "alexis";
    $swag = "infinity";
    proclaimSwagness();
  }

  public function proclaimSwagness() {
    echo "swag on " + $this->swag
  }

}

$alexis = new Alexis();
?>

File: alexis.cs
using System;

namespace Alexis {
  class Alexis {
    public string Name { get; set; }; 
    public string Swag { get; set; };

    public Alexis() {
      Name = "alexis";
      Swag = "infinity";
      ProclaimSwagness();
    }

    public void ProclaimSwagness() {
      Console.WriteLine("swag on {0}", Swag);
    }
    public static void Main(string[] args) {
      var alexis = new Alexis();
    }
  }
}